week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,9 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
+  return (typeof num === "number" && !excludedNums.includes(num) && Number.isInteger(num))?true:false;
```

Comment:
Again another nice use of a ternary operator. But the same advice from above about small well-named functions applies here too.